tests/cli/main.cpp

```diff
@@ -221,6 +221,34 @@ BOOST_AUTO_TEST_CASE( cli_connect )
    app1->shutdown();
 }
 
+////////////////
+// Start a server and connect using the same calls as the CLI
+// Quit wallet and be sure that file was saved correctly
+////////////////
+BOOST_AUTO_TEST_CASE( cli_quit )
```

Comment:
This test does not validate the wallet file was saved correctly.
1. Do we need to validate the wallet file? And if so...
2. What is the best way to validate it?

Reply:
one idea i think i posted before for this will be something like:
- connect to wallet.
- create an account.
- quit.
- connect to wallet.
- check if created account is there. 

however it seems that the last version of the `quit` command is not saving the wallet.

Reply:
in cli_wallet::quit command we don't save file, should we save it ?

Reply:
in my opinion yes but i am not following this issue very close in the last time, not sure if it was decided to just don't save.

Reply:
we need to decide what is the best way, save or not to save

Reply:
Exiting cleanly should save the wallet file. Exiting with CTRL+D saves it. This should save it as well. I haven't looked, but I would imagine that the CTRL+D functionality would call a function, and the quit() functionality should call the same function.

Reply:
As discussed above, we need to save wallet file. The following code will save the wallet file: https://github.com/bitshares/bitshares-core/blob/4a2f333444b5d9a0cd13c78a16393594e9db1e6d/programs/cli_wallet/main.cpp#L296

Reply:
IMHO we can merge this PR and create a new issue about how to validate the wallet file.

Reply:
ok sure ! will do save wallet file and will improve test, soon will make commit !

Reply:
absolutely agree about new issue about validating how the wallet file was saved

Reply:
Guys, please discuss the wallet file saving issue in https://github.com/bitshares/bitshares-core/issues/1109. Thanks.